lib/twingly/url.rb

```diff
@@ -100,6 +100,10 @@ def tld
       public_suffix_domain.tld
     end
 
+    def ttld
```

Comment:
> What would the comment say? Wouldn't it be better to call the method true_tld then?

Nah, I don't think true is any better. "True" doesn't add any context when to use this.

> @jage Like this?

Yes, that's better.